src/library_syscall.js

```diff
@@ -808,7 +808,8 @@ var SyscallsLibrary = {
     // hack to support printf in NO_FILESYSTEM
     var stream = SYSCALLS.get(), iov = SYSCALLS.get(), iovcnt = SYSCALLS.get();
     var ret = 0;
-    if (!___syscall146.buffers) {
+#if USE_PRINTCHARS_FOR_STDOUT == 0    
+    if (!___syscall146.buffer) {
```

Comment:
This looks like has dropped `s` at the end of the if? `buffers` instead of `buffer`?

Reply:
Btw, does `#if !USE_PRINTCHARS_FOR_STDOUT` work?